src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -9,7 +9,11 @@ import com.typesafe.sbt.git.ReadableGit
 import com.typesafe.sbt.git.DefaultReadableGit
 
 /** This plugin has all the basic 'git' functionality for other plugins. */
-object SbtGit extends Plugin {
+object SbtGit extends AutoPlugin {
```

Comment:
We should be careful with breaking naming conventions. Are you (except for this case) recommending to use FooPlugin instead of SbtFoo (which was the convention so far)?